IP_POLICY_GUIDANCE.md

```diff
@@ -0,0 +1,98 @@
+# OpenJS Foundation IP Policy Guidance
+
+The purpose of this guide is to make it straightforward for projects
+to follow the [OpenJS Foundation IP Policy][IP Policy].
+
+Of course, the source of truth remains the [IP Policy][].
+
+If in doubt about anything related to IP after reading this document,
+first read the [IP Policy][], and then [ask for help](#getting-help).
+
+## Basic requirements
+
+There are four key requirements for all OpenJS Foundation projects:
+
+1. [Licensing](#1-licensing)
+2. [Copyright notices](#2-copyright-notices)
+3. Displaying the [standard website footer](#3-standard-website-footer) on the project's website
+4. [Adopting the DCO or a CLA](#4-adopting-the-dco-or-a-cla)
+
+### 1. Licensing
+
+Code may be licensed under:
+  * [Apache, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
+  * [MIT](https://opensource.org/licenses/MIT)
+  * [2-Clause BSD](https://opensource.org/licenses/BSD-2-Clause)
+  * [3-Clause BSD](https://opensource.org/licenses/BSD-3-Clause)
+
+Documentation may be licensed under:
+  * [CC BY 4.0](http://creativecommons.org/licenses/by/4.0/)
+  * [MIT](https://opensource.org/licenses/MIT)
+
+If you need to use a license which isn't in the [IP Policy][], you will
+[need to obtain permission from the Board to do so](#requiring-an-exemption-from-the-board).
+
+### 2. Copyright notices
+
+The recommendation is to use a general copyright statement
+of the following form (where XYZ is the project's name):
+
+- **Copyright The XYZ Authors.**
+- **Copyright The XYZ Contributors.**
+- **Copyright Contributors to the XYZ project.**
+
+By using this format, the project avoids having to deal with
+names of copyright holders, years or ranges of years,
+and variations on the (c) symbol.
+
+_Please note that you **must not** change or remove existing copyright lines
+unless you put them there **and** have the right to do so.
+If there are existing copyright lines
+add the recommended copyright statement below the existing copyright lines.
+You may however update copyright lines that say
+"JS Foundation" or "Node.js Foundation" to "OpenJS Foundation"
+as this simply reflects the merged status of the organizations._
+
+### 3. Standard website footer
+
+All project websites need to have a [standard website footer][].
+
+### 4. Adopting the DCO or a CLA
+
+Each project must adopt either the [Developer Certificate of Origin (DCO)][DCO]
```

Comment:
I disagree that this should be the policy, but this PR isn’t imposing it, merely documenting it, so i won’t derail on that here :-)

Reply:
Yeah, thanks for your understanding.